examples/vpc-with-dns/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to provision resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "us-south"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "dns"
+}
+
+variable "name" {
+  description = "The name of the vpc"
+  type        = string
+  default     = "sz-vpc"
```

Comment:
Do you mean `slz`?

Reply:
Yeah I meant slz. This is a typo from my end. Thank you for pointing. I have corrected it.